data/root/etc/inst_setup

```diff
@@ -211,7 +211,7 @@ create_zram_swap_disable_hook() {
 #! /bin/sh -x
 
 # at least 2 swap devices
-if [ "\$(wc -l < /proc/swaps)" != 2 ] ; then
+if [ -b "$zram_swap_dev" ] && [ "\$(wc -l < /proc/swaps)" != 2 ] ; then
```

Comment:
I think the critical point is

https://github.com/openSUSE/installation-images/blob/master/data/root/etc/inst_setup#L184

which also needs quotes.

Reply:
I overlooked that one, thanks!